### PR TITLE
fix(mobile): restructuring of config nav section

### PR DESCRIPTION
**Pull request by nwhite222**
Relates to ongoing mobile edits

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -88,3 +88,5 @@ yarn-error.log
 .devcontainer
 
 src/images/infrastructure_screenshot_full_sonarqube-dashboard.webp
+src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx
+src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx
```
Comment (ally-sassman) [inline .gitignore:92]: @nwhite222 I'm not sure if these code lines are supposed to be here - what was this for again?

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx [hunk 2]
```diff
@@ -4,6 +4,11 @@ tags:
   - Mobile monitoring
   - API guides
 metaDescription: 'Customize your mobile agent instrumentation to send custom data and metrics to New Relic.'
+redirects:
+  - /node/13986
+  - /node/13991
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx:9]: @nwhite222 What are these for?

**Review by ally-sassman — DISMISSED** [review]
Approved but see my questions on two file changes.

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx (hunk 2 above)
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx:9]:
```suggestion

```

Comment (nwhite222) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx]: I'm not sure...I think those were added automatically. I can see if anything happens when I delete them.

File: .gitignore (hunk 1 above)
Comment (nwhite222) [inline .gitignore:92]:
```suggestion

```

Comment (nwhite222) [reply .gitignore:92]: I'm also not sure what happened here, but I'm going to try deleting them.

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**